app/models/concerns/updatable.rb

```diff
@@ -0,0 +1,44 @@
+module Updatable
+  extend ActiveSupport::Concern
+
+  included do
+    def updated
+      public_updated_at if any_updates?
+    end
+
+    def history
+      return [] unless any_updates?
+
+      reverse_chronological_change_history
+    end
+  end
+
+private
+
+  def any_updates?
+    if public_updated_at && initial_publication_date
+      Time.zone.parse(public_updated_at) != Time.zone.parse(initial_publication_date)
+    else
+      false
+    end
+  end
+
+  def initial_publication_date
+    first_public_at || first_published_at
+  end
+
+  def reverse_chronological_change_history
+    change_history.sort_by { |item| Time.zone.parse(item[:timestamp]) }.reverse
+  end
+
+  def change_history
+    changes = content_store_hash.dig("details", "change_history") || []
+    changes.map do |item|
+      {
+        display_time: item["public_timestamp"],
```

Comment:
This is missing the `display_date` method from https://github.com/alphagov/government-frontend/blob/a938e4bee78b794e4e5303a1bde1dfd05e94b287/app/presenters/content_item/updatable.rb#L31, or is the technical approach going forward to use it in the view? e.g. we should be including it [here](https://github.com/alphagov/government-frontend/blob/af54031948650448b965ac274c3248d23658c4f6/app/views/components/_published_dates.html.erb#L37) instead